iceoryx_posh/include/iceoryx_posh/internal/roudi/roudi.hpp

```diff
@@ -17,6 +17,7 @@
 #ifndef IOX_POSH_ROUDI_ROUDI_MULTI_PROCESS_HPP
 #define IOX_POSH_ROUDI_ROUDI_MULTI_PROCESS_HPP
 
+#include "iceoryx_hoofs/cxx/convert.hpp"
```

Comment:
I think this is required only in `roudi.cpp` therefore I would move it into the cpp file. This reduces a bit the compile time and makes the header dependencies more clear.